docs/manuals/mapping.md

```diff
@@ -1,15 +1,19 @@
 # Mapper Frontend Documentation
 
-The **Mapper Frontend** was developed to provide a simpler, more intuitive
+The **Mapper's page** was developed to provide a simpler, more intuitive
 mapping experience.
 
 ## Part (A): Setup For Mapping
 
 ### Step 1: Install The Custom ODK Collect Mobile App
 
 - The first time you load the project, you should be prompted to download
-  the custom `.apk` from the sidebar.
+  the custom `.apk` from the upper right corner of your browser window
+  usually represented by three vertical dots.
```

Comment:
It's not `dot`. It shall be represented by a hamburger menu(might be technical)/menu/☰